src/user-guide/removing-workflows.rst

```diff
@@ -0,0 +1,88 @@
+.. _Removing-workflows:
+
+Removing Workflows
+==================
+
+If you want to delete an installed workflow from the :term:`cylc-run directory`
+(or just delete certain subdirectories), the recommended way is using
+the ``cylc clean`` command. ``cylc clean`` takes care of deleting workflows
+on the local filesystem was well as any remote install targets. It also
+follows any symlink directories specified in
+:cylc:conf:`global.cylc[install][symlink dirs]`
+(but will never follow other symlink directories). It will not remove the
+workflow :term:`source directory` either.
```

Comment:
A couple of points - 
- Hemmingway Editor gave this a US reading grade of 16, I managed to get it down to 11 with

> To delete an installed workflow (or run) from the cylc run directory use cylc clean. Cylc clean removes files from remote install targets as well as local file systems. It also follows any symlink directories specified in global.cylc[install][symlink dirs] (but will never follow other symlink directories). It does not remove the workflow source directory either.

- I couldn't streamline the sentence sentence starting "It also follows any..."
- At the risk of pointing my own flaws, I think all sentences here should be in the same tense. I'd suggest the "will" in the last sentence be "does"